tests/test_ac_write_path.py

```
@@ -926,6 +926,136 @@ async def test_reraises_domain_error_unchanged(self) -> None:
+        ]
+        return appliance_switches, command
+
+    # The air conditioner's `settings` as the AS35PBPHRA-PRE declares the part
+    # that matters here: the toggled flag, a louvre parameter the sanitizer has to
+    # repair, and two unrelated siblings that must still travel.
+    _AC_SETTINGS = {
+        "lightStatus": Param("0"),
+        "windDirectionVertical": Param("0", values=["2", "4", "8"]),
+        "machMode": Param("1"),
+        "tempSel": Param("22"),
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)